conventions/views.py

```diff
@@ -56,12 +57,36 @@
 
 
 @login_required
-def index(request):
-    result = conventions_index(request)
+@require_GET
+def search(request, category: str):
+    only_active = category == 'en-cours'
+    query_set = request.user.conventions(only_active)
```

Comment:
Est-ce possible de l'écrire comme ça ?

```suggestion
    query_set = request.user.conventions(active=only_active)
```

Reply:
Bien sûr !